src/schema/nmdc.yaml

```diff
@@ -1043,19 +1050,21 @@ slots:
     range: WorkflowExecution
     description: The identifier for the analysis activity that generated the functional annotation results, where the analysis activity is an instance of the/an appropriate subclass of WorkflowExecution
     required: true
-    any_of:
-      - range: MetagenomeAnnotation
-      - range: MetatranscriptomeAnnotation
+    deprecated: "not used. 2024-10 https://github.com/microbiomedata/nmdc-schema/issues/1253"
+
 
   gene_function_id:
     range: uriorcurie
     description: The identifier for the gene function.
     examples:
       - value: KEGG.ORTHOLOGY:K00627
     required: true
+
   count:
     range: integer
+    description: The number of sequences (for a metagenome or metatranscriptome) or spectra (for metaproteomics) associated with the specified function.
```

Comment:
<3

Reply:
I would like to move that definition for count to `slot_usage`. The `count` slot could easily be used with different meanings in other class contexts.